system/bluetooth_monitor/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.5)
+project(bluetooth_monitor)
+
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
```

Comment:
Please use `autoware_cmake` #849.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/862/commits/704c0075a0c2d4523a08df2eea78f2942d6eacb4